frontend/apps/reader/readerui.lua

```diff
@@ -329,6 +330,7 @@ end
 
 function ReaderUI:showReader(file)
     DEBUG("show reader ui")
+    ReadHistory:addItem(file)
```

Comment:
Yeah, I actually prefer less number like 250. UI-wise, our menu widget does lazy rendering, so it handle very large list without issue. For me, I only use history menu for fast switching between the books that I am currently reading. Since I don't read many books at once, I never go pass the first page ;P